docs/docs/hive.md

```diff
@@ -300,6 +300,7 @@ The result is:
 | i                                  | BUCKET\[2\]    | NULL
 
 The supported transformations for Hive are the same as for Spark:
+
```

Comment:
Is this needed?

Reply:
Yes, the generated mkdocs page is broken as I pasted in the PR description.